src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -218,9 +218,7 @@
     <PlatformManifestFileEntry Include="icudt_optimal.dat" IsNative="true" />
     <PlatformManifestFileEntry Include="icudt_optimal_no_CJK.dat" IsNative="true" />
     <PlatformManifestFileEntry Include="runtime.iffe.js" IsNative="true" />
-    <PlatformManifestFileEntry Include="binding_support.js" IsNative="true" />
-    <PlatformManifestFileEntry Include="dotnet_support.js" IsNative="true" />
-    <PlatformManifestFileEntry Include="library_mono.js" IsNative="true" />
+    <PlatformManifestFileEntry Include="library-dotnet.js" IsNative="true" />
```

Comment:
This is emcc input, not output. The output is single file like `\artifacts\bin\microsoft.netcore.app.runtime.browser-wasm\Release\runtimes\browser-wasm\native\dotnet.js` which they rename to like `dotnet.6.0.0-rc.1.21430.1.js` on blazor side. That would not change for them. I tested it works.

Reply:
The difference is that the release version is now pre-minified by us. And therefore not so easy to debug.

On the other hand, the debug version never worked for Blazor even before, because they use `_malloc` before the runtime is initialized and the debug version of emcc runtime throws on it.

I thought that we need to fix our API first and then fix Blazor startup to use it properly. Then we could include debug version of js files and improve the compilation tooling to pick it in debug mode files instead of minified one.